lib/cherry_silt/connection.rb

```diff
@@ -56,24 +52,29 @@ def receive_data(data)
   end
 
   def unbind
+    puts 'Bye Bye'
+    # this doesn't seem to be dropping connection
     @server.connections.delete(self)
```

Comment:
I think in addition to deleting the connection from the server objects connections array, we may need to also make sure the connection is killed somehow.  I'll do some testing and get back on it.